reader.py

```diff
@@ -0,0 +1,35 @@
+import csv
+import os
+import sys
+
+
+class Reader:
+
+    weather_records = []
+    weather_attributes = ["PKT", "Max TemperatureC", "Min TemperatureC", "Max Humidity", " Mean Humidity"]
+
+    def read_weather_monthly_record(self, weather_monthly_record_file_path):
+        with open(weather_monthly_record_file_path) as weather_monthly_record:
+            try:
+                weather_daily_records = csv.DictReader(weather_monthly_record)
+                # print(weather_daily_records.keys())
+                for single_day_weather_record in weather_daily_records:
+                    try:
+                        date = single_day_weather_record['PKT']
+                    except KeyError:
+                        date = single_day_weather_record['PKST']
+                    single_day_weather_record = [date,
+                                                 single_day_weather_record['Max TemperatureC'],
+                                                 single_day_weather_record['Min TemperatureC'],
+                                                 single_day_weather_record['Max Humidity'],
+                                                 single_day_weather_record[' Mean Humidity']]
```

Comment:
```
single_day_weather_record = [
    date,
    single_day_weather_record['Max TemperatureC'],
    single_day_weather_record['Min TemperatureC'],
    single_day_weather_record['Max Humidity'],
    single_day_weather_record[' Mean Humidity']
]
```

Reply:
We can directly append it to weather records instead of wasting a variable space.

Reply:
Sure